v2/src/components/snippetConfigForm/index.tsx

```diff
@@ -119,13 +119,13 @@ export default class SnippetConfigForm<T extends keyof any> extends React.PureCo
               if (typeof c === "string") {
                 for (const [ind, question] of this.props.questions.entries()) {
                   const value = this.state.answers[ind];
-                  c = c.replaceAll(`^{form_${question.id}}`, value);
+                  c = c.split(`^{form_${question.id}}`).join(value);
```

Comment:
We actually have a pollyfill for replace all that we use: https://github.com/supertokens/docs/blob/master/v2/src/plugins/copyDocsAndCodeTypeChecking.js#L11

You should use that, or copy that function and use that?

Reply:
updated